### ISSUE TITLE
Research why uname -s is inconsistent in Windows jobs

### ISSUE THREAD

**Issue by MMulthaupt** [setlog/trivrost#110]
`uname -s` sometimes reports `MSYS_NT` and sometimes `MING64_NT` on the Windows runner. One should expect all runners are configured the same.

**Comment by MMulthaupt** [comment]
I suspect this changes depending on whether or not the `shell: bash`-keyvalue is added to a step.

**Comment by dragetd** [comment]
Yeah, MINGW64 is a cygwin thing. I was expecing that list to be non-exhaustive. Tho I am pretty sure that problem will go away with mage.
